Gemfile.lock

```diff
@@ -209,60 +209,60 @@ GEM
       sprockets-rails
       tilt
     semantic_range (2.3.0)
-    spring (2.1.0)
+    spring (2.1.1)
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
     sprockets (4.0.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.2.1)
+    sprockets-rails (3.2.2)
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    tinymce-rails (5.4.1)
+    tinymce-rails (5.6.1)
```

Comment:
### [tinymce-rails](https://github.com/spohlenz/tinymce-rails)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 5.4.1 → 5.6.1

[_~~change-log~~, [source-code](https://github.com/spohlenz/tinymce-rails)_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/spohlenz/tinymce-rails/compare/v5.4.1...v5.6.1).

These are the individual commits:
- (4b4c343) [Use SVG format for Travis badge](https://github.com/spohlenz/tinymce-rails/commit/4b4c343f7ddbc04f5a9a251bfc8c3552c0557f1a)
- (b86fdc8) [Add bundler tasks to Rakefile](https://github.com/spohlenz/tinymce-rails/commit/b86fdc8845b345eb86b9677e0342a83a6f7b71be)
- (2976cd1) [Update to TinyMCE 5.5.1](https://github.com/spohlenz/tinymce-rails/commit/2976cd123683145dddfeac757c6c141af6ae2e41)
- (f8f286f) [Add vendored TypeScript file](https://github.com/spohlenz/tinymce-rails/commit/f8f286f153a3c357356ad38a59274ff397a78b23)
- (67a28e0) [Update to TinyMCE 5.6.0](https://github.com/spohlenz/tinymce-rails/commit/67a28e078578c26363c13d8c07087c309d994511)
- (5a82e32) [Update to TinyMCE 5.6.1](https://github.com/spohlenz/tinymce-rails/commit/5a82e32f29cc066cee8290fbf5514925f379653d)


</details>